sdcm/cluster_k8s/__init__.py

```diff
@@ -2083,7 +2083,8 @@ def __init__(self,
 
     get_scylla_args = cluster_docker.ScyllaDockerCluster.get_scylla_args
 
-    def wait_for_nodes_up_and_normal(self, nodes=None, verification_node=None):
+    def wait_for_nodes_up_and_normal(self, nodes=None, verification_node=None, iterations=None, sleep_time=None,
+                                     timeout=None):  # pylint: disable=too-many-arguments
```

Comment:
Shouldn't we also update the "check_nodes_up_and_normal" method defined on the line 2093 here?
By setting there new parameters with old values as defaults?

Reply:
Base method do not need it. I don't want to change base method and add new parameters

Reply:
As of now it doesn't get applied, so that is the reason I asked for the update.
Not having effect why then we apply this change at all?

Reply:
Because I changed base methos and add new parameters, I had to add those parames to override methods, despite it's not needed there. So it suppose not to be in use

Reply:
Ok, let it be so.